Flaky test_submit

**Describe the bug**

On the third retry, test_submit failed (See this issue for previous runs: https://github.com/equinor/ert/issues/7310)

```
integration_tests/scheduler/test_lsf_driver.py::test_submit 
[gw2] FAILED integration_tests/scheduler/test_lsf_driver.py::test_submit 
```

```
=================================== FAILURES ===================================
_________________________________ test_submit __________________________________
[gw2] linux -- Python 3.8.18 /tmp/f_scout_ci/actions-runner-01/_temp/test-kenv/root/bin/python

item = <Coroutine test_submit>

    @pytest.hookimpl(wrapper=True, tryfirst=True)
    def pytest_runtest_call(item):
>       result = yield

../../test-kenv/root/lib64/python3.8/site-packages/pytestqt/plugin.py:189: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
../../test-kenv/root/lib64/python3.8/site-packages/pytest_asyncio/plugin.py:440: in runtest
    super().runtest()
../../test-kenv/root/lib64/python3.8/site-packages/pytest_asyncio/plugin.py:907: in inner
    _loop.run_until_complete(task)
/opt/rh/rh-python38/root/usr/lib64/python3.8/asyncio/base_events.py:603: in run_until_complete
    self.run_forever()
/opt/rh/rh-python38/root/usr/lib64/python3.8/asyncio/base_events.py:570: in run_forever
    self._run_once()
/opt/rh/rh-python38/root/usr/lib64/python3.8/asyncio/base_events.py:1823: in _run_once
    event_list = self._selector.select(timeout)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <selectors.EpollSelector object at 0x7fe8306e9ee0>, timeout = -1

    def select(self, timeout=None):
        if timeout is None:
            timeout = -1
        elif timeout <= 0:
            timeout = 0
        else:
            # epoll_wait() has a resolution of 1 millisecond, round away
            # from zero to wait *at least* timeout seconds.
            timeout = math.ceil(timeout * 1e3) * 1e-3
    
        # epoll_wait() expects `maxevents` to be greater than zero;
        # we want to make sure that `select()` can be called when no
        # FD is registered.
        max_ev = max(len(self._fd_to_key), 1)
    
        ready = []
        try:
>           fd_event_list = self._selector.poll(timeout, max_ev)
E           Failed: Timeout >5.0s

/opt/rh/rh-python38/root/usr/lib64/python3.8/selectors.py:468: Failed
```

Logs: https://github.com/equinor/komodo-releases/actions/runs/8088980330/job/22124453157

**To reproduce**
TBD

**Expected behaviour**
Not fail

**Screenshots**
N/A

**Environment**
 - OS: RHEL7
 - ERT/Komodo release: Bleeding
 - Python version: 3.8
 - Remote/HPC execution involved: [yes|no]

**Additional context**
N/A

--- a/tests/integration_tests/scheduler/test_lsf_driver.py
+++ b/tests/integration_tests/scheduler/test_lsf_driver.py
@@ -44,7 +44,6 @@ async def poll(driver: Driver, expected: Set[int], *, started=None, finished=Non
         poll_task.cancel()
 
 
-@pytest.mark.timeout(5)
 @pytest.mark.integration_test
 async def test_submit(tmp_path):
     driver = LsfDriver()
@@ -71,7 +70,6 @@ async def test_submit_something_that_fails():
     assert finished_called
 
 
-@pytest.mark.timeout(5)
 async def test_kill():
     driver = LsfDriver()
     aborted_called = False

